content/games/factions/routines.md

```diff
@@ -0,0 +1,158 @@
+
+- Bold and Ferocious Companies prefer to be the attackers.
+- Cautious and Guarded Companies prefer to be the defenders.
+- Arcane Companies prefer whichever role is more eldritch (e.g. the ritualists in the Ritual
+  scenario). If unclear, they choose at random.
+
+### Deploying
+
+Routine Companies deploy in accordance with the scenario, but with some alterations. Define four
+quadrants on the table that include the valid deployment zones for the Routine Company in the
+scenario. Follow this process:
+
+1. Line up the Routine Company's Groups from lowest to highest point value along one edge of the
+   table.
+1. To deploy the first Group, roll 1d6:
+   - If the result is _even_, deploy the group farthest right in the line to the nearest deployment
+     zone.
+   - If the result is _odd_, deploy the group farthest left in the line to the farthest deployment
+     zone.
+1. For each remaining Group, roll 1d6:
+   - If the result is odd, select the left-most Group. If even, select the right-most Group.
+   - If the result is _lower_ than the previous roll, deploy the selected Group in an adjacent
+     quadrant (chosen at random).
+   - If the result is _equal_ to the previous roll, deploy in the same quadrant.
+   - If the result is _higher_ than the previous roll, deploy in the opposite quadrant.
+1. Adjust the Company's deployment based on their personality, as needed:
+   - Arcane Companies must deploy Groups with the Casting or Summoning order within 6" of a friendly
+     group, if possible.
+   - Bold Companies must deploy their Groups as close an enemy or objective as possible.
+   - Cautious Companies must deploy their Groups with Shooting, Casting, or Summoning orders within
+     6" of a friendly group and as far away from enemy Groups as possible.
+   - Ferocious Companies must deploy their Groups within range to Attack or Shoot, if possible.
+   - Guarded Companies must deploy their groups within 6" of a friendly Group. If any Groups have
+     the Defensive trait, they must be as evenly dispersed among Group clusters as possible.
+   - Whenever a decision is unclear, choose randomly.
+
+### Ambitions
+
+Routine Companies may have [Ambitions][1], based on their Personality. For each Ambition listed next
+to the Routine Company's Personality below, roll 1d6. On a 4+, they have that Ambition. Routine
+Company Ambitions always count as revealed.
+
+- **Arcane:** Give Better Than We Get, Draw First Blood,They Will Know My Name.
+- **Bold:** They Will Reel Before Us, Give Better Than We Get, Draw First Blood, They Will Know My
+  Name.
+- **Cautious:** We Will Not Break, Give Better Than We Get, Rain Hell Upon Them.
+- **Ferocious:** They Will Reel Before Us, We Will Snap Their Spine, Break Even Their Anvil, Draw
+  First Blood, We Will All Shed Blood.
+- **Guarded:** We Will Not Break, Win Without Moving, Do Not Waver, Give Better Than We Get, Rain
+  Hell Upon Them.
+- **Incompetent:** None.
```

Comment:
```suggestion
- **Incompetent:** _None_.
```

Reply:
to technically differentiate from a trait title (so it doesn't look like they should go looking for a trait titled "None").